app/managers.py

```diff
@@ -0,0 +1,38 @@
+from models import Actor
+
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
```

Comment:
The database connection is opened here but not closed anywhere in the class. Consider implementing a method to close the connection when it's no longer needed to prevent resource leaks.